components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2176,4 +2176,49 @@ void RewardsServiceImpl::SetShortRetries(bool short_retries) {
   bat_ledger_service_->SetShortRetries(short_retries);
 }
 
+bool SavePendingContributionOnFileTaskRunner(PublisherInfoDatabase* backend,
+    const ledger::PendingContributionList& list) {
+  if (!backend) {
+    return 0;
+  }
+
+  return backend->InsertPendingContribution(list);
+}
+
+void RewardsServiceImpl::OnSavePendingContribution(bool result) {
+  // TODO(nejczdovc) add callback with db result
+  LOG(ERROR) << "OnSavePendingContribution" << result;
```

Comment:
Any action needed here? or is that with phase 2?

Reply:
no action currently needed, will be done in phase 2